src/khoj/interface/web/content_type_input.html

```diff
@@ -44,9 +44,9 @@ <h2 class="section-title">
                 </tr>
             </table>
 
-            <h4>You probably don't need to edit these.</h4>
+            <h4 style="display: none;">You probably don't need to edit these.</h4>
```

Comment:
Should we just delete this line? Like you've done in the `processor_conversation_input.html`